portal/templates/admin/patients_substudy.html

```diff
@@ -95,7 +95,13 @@ <h2>{{list_title}}</h2>
                       </td>
                       <td>{% for org in patient.organizations | sort(attribute='id') %}<span class="medium-text" data-org-id="{{org.id}}">{{org.name}}</span><br/>{% endfor %}</td>
                       <td>{{patient.clinician if patient.clinician else ""}}</td>
-                      <td>{{patient.action_state}}</td>
+                      <td>
+                        {% if patient.action_state == "required" or patient.action_state == "due" or patient.action_state == "overdue" %}
```

Comment:
This is a bit more succinct
```suggestion
                        {% if patient.action_state in ("required", "due", "overdue") %}
```